src/Core/Enums/PlanType.cs

```diff
@@ -20,12 +20,20 @@ public enum PlanType : byte
     Custom = 6,
     [Display(Name = "Families")]
     FamiliesAnnually = 7,
+    [Display(Name = "Teams (Monthly) 2023")]
+    TeamsMonthly2020 = 8,
+    [Display(Name = "Teams (Annually) 2023")]
+    TeamsAnnually2020 = 9,
+    [Display(Name = "Enterprise (Monthly) 2023")]
+    EnterpriseMonthly2020 = 10,
+    [Display(Name = "Enterprise (Annually) 2023")]
+    EnterpriseAnnually2020 = 11,
```

Comment:
Something to double-check, the display names for the `2020` plans are currently `2023`.

In the static store plans, the names use `2020`. Disregard if this intended.